lib/freeathome.js

```diff
@@ -101,7 +114,11 @@ module.exports = class FreeAtHomeApi {
                     if (value != undefined) {
                         that.adapter.setObjectNotExists(serialNumber + '.' + channel + '.' + key, {
                             type: 'state',
-                            common: {},
+                            common: {
+                                name: key + ' value',
+                                read: true,
+                                write: false
```

Comment:
This one is tough, I assume it could be of different types, depending on the device. I don't know what to specify here.